Author: Pino Toscano <pino@debian.org>
Description: fix literal strings for string printf-like format
 Make the text buffers const, so they can be properly handled as string
 literals for snprintf().
Last-Update: 2011-09-29
Forwarded: no

---
 tools/compiler.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/tools/compiler.c
+++ b/tools/compiler.c
@@ -266,9 +266,9 @@ double value;
 static int
 next_token ()
 {
-	static char toomv[] = "More than %d variables";
-	static char toomc[] = "More than %d constants";
-	static char badop[] = "Illegal operator";
+	static const char toomv[] = "More than %d variables";
+	static const char toomc[] = "More than %d constants";
+	static const char badop[] = "Illegal operator";
 	int tok = ERR;	/* just something illegal */
 	char c;
 
